src/CoinbaseV3.lua

```diff
@@ -0,0 +1,236 @@
+-- Inofficial Coinbase Extension (www.coinbase.com) for MoneyMoney
+-- Fetches balances from Coinbase API V3 and returns them as securities
+--
+-- Username: Coinbase Key Name
+-- Password: Coinbase Private Key
+--
+-- Copyright (c) 2024 Felix Nensa
+-- Copyright (c) 2020-2022 Martin Wilhelmi
+-- Copyright (c) 2017 Nico Lindemann
+--
+-- Permission is hereby granted, free of charge, to any person obtaining a copy
+-- of this software and associated documentation files (the "Software"), to deal
+-- in the Software without restriction, including without limitation the rights
+-- to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+-- copies of the Software, and to permit persons to whom the Software is
+-- furnished to do so, subject to the following conditions:
+--
+-- The above copyright notice and this permission notice shall be included in all
+-- copies or substantial portions of the Software.
+--
+-- THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+-- IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+-- FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+-- AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+-- LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+-- OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+-- SOFTWARE.
+
+WebBanking {
+    version = 2.0,
```

Comment:
Would be nice if, this version can be just a successor of version 2 because then I will not lose my history.

so my suggestion is keep `service`and `bankCode` to `Coinbase Account` as it was before and bump version to `3.0` instead